region/closedsource/docker-compose.yml

```
@@ -10,6 +10,31 @@ networks:
               gateway: "${DOCKER_MGMT_IP6}1"
 
 services:
+   flower:
+      image: mher/flower:0.9.5
+      container_name: flowerregion
+      networks:
+         mgmt:
+            ipv6_address: "${DOCKER_MGMT_IP6}6002:4"
+      ports:
+         - "${PMS6}:5555:5555"
+      entrypoint: flower --port=5555 --broker=amqp://rabbitmq:5672/
+
+   jaeger:
+      image: jaegertracing/jaeger-agent:latest
+      container_name: jaeger-agent
+      command: ["--reporter.grpc.host-port=collector.${REGION_PAM_NAME}.${REGION_PAM_ORGANIZATION_URL}:14250"]
```

```suggestion
      command: ["--reporter.grpc.host-port=collector.${PAM_NAME}.${PAM_ORGANIZATION_URL}:14250"]
```

As these variables are in the .env file for a COP, should we use the same name for regions? It would mean a change to the ISO, to move to the "common" section for all PODs but this naming also requires a change to a ISO

---

I think so. Should I open a dedicated MR for removing some duplicated variables?

---

Please, one of the areas I want to work on next is
1. Reviewing the ISO and streamlining the naming conventions
2. Implement the naming conventions across the CloudCIX docker containers, fields in the PAT application/app and the compose files.